wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -1300,6 +1327,184 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>			
+			<xs:element name="AddAudioClipResponse">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Token" type="tt:ReferenceToken">						
+							<xs:annotation>
+								<xs:documentation>Unique Token associated with the audio clip.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element name="UploadUri" type="xs:anyURI">						
+							<xs:annotation>
+								<xs:documentation>An URL to which the audio clip may be uploaded.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element name="ExpiryTime" type="xs:dateTime">						
+							<xs:annotation>
+								<xs:documentation>Expiry time before which client should upload audio clip to the device.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>				
+			</xs:element>
+			
+			<xs:element name="SetAudioClip">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="AudioClipConfiguration" type="tr2:AudioClipConfiguration">
+							<xs:annotation>
+								<xs:documentation>Audio Clip Configuration to modify.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>			
+			<xs:element name="SetAudioClipResponse">
+				<xs:complexType>
+					<xs:sequence>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+				
+			<xs:element name="DeleteAudioClip">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Token" type="tt:ReferenceToken">
+							<xs:annotation>
+								<xs:documentation>Token of the audio clip to be deleted from the device.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>			
+			<xs:element name="DeleteAudioClipResponse">
+				<xs:complexType>
+					<xs:sequence>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+
+			<xs:element name="PlayAudioClip">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="Token" type="tt:ReferenceToken">
+							<xs:annotation>
+								<xs:documentation>Unique Token associated with the audio clip.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element name="AudioOutputToken" type="tt:ReferenceToken" minOccurs="0">
+							<xs:annotation>
+								<xs:documentation>Optional Token of the physical Audio Output. If the AudioOutputToken is not provided in the API request, the AudioOutputToken in AudioClipConfiguration to be used.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element name="Play" type="xs:boolean">
+							<xs:annotation>
+								<xs:documentation>If it is true play the audio clip, if it is false stop ongoing audio clip.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element name="RepeatCycles" type="xs:int" minOccurs="0">
```

Comment:
What is the use case of having RepeatCycles stored in config? 

If able to override RepeatCycles, what about Gain?